style.css

```diff
@@ -78,8 +83,187 @@ body {
   color: #505f79;
 }
 
+li {
+  list-style-type: none;
+}
+
 .social-icons li {
   list-style-type: none;
   padding: 0;
   margin: 0;
 }
+
+/* works section */
+
+.works {
+  display: grid;
+  gap: 25vw;
+  padding: 0 1vw;
+  margin-top: 25vw;
+  margin-bottom: 36vw;
+}
+
+.card {
+  width: 90vw;
+  margin: auto;
+  background-color: white;
+  padding: 2vw;
+  border: 1px solid #dfe1e6;
+  border-radius: 1rem;
+  -webkit-border-radius: 1rem;
+  -moz-border-radius: 1rem;
+  -ms-border-radius: 1rem;
+  -o-border-radius: 1rem;
+  display: flex;
+  flex-direction: column;
+  gap: 4vw;
+  padding-bottom: 5vw;
+}
+
```

Comment:
- Kindly Make Sure Stick to the design Margins as much as possible to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design, according to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma). 

**Original Design**
![Screenshot from 2023-04-28 12-20-24](https://user-images.githubusercontent.com/97834160/235123560-b7692228-dac3-45ce-9db0-72d3e8d10ce5.png)

**Your Design**
![Screenshot from 2023-04-28 12-19-47](https://user-images.githubusercontent.com/97834160/235123654-5292ca46-75b4-493a-8fcd-4966221252a1.png)